openadapt/scripts/reset_db.py

```diff
@@ -1,16 +1,15 @@
 import os
 from subprocess import run, PIPE
-from openadapt.config import getenv_fallback
+from openadapt import config
 
 
 def reset_db():
     """
     The function clears the database by removing the database file and running a
     database migration using Alembic.
     """
```

Comment:
A new line will be great as per the styling convention we are following.

Reply:
Oh I was unaware of this since most styles are against newlines after method docstrings